client/src/components/Tags.tsx

```diff
@@ -28,7 +28,7 @@ const Tags = () => {
 		console.log(id, 'YOU CLICKED!!');
 		try {
 			await axios.patch(`/api/tags/${id}`);
-			getUserTags();
+			await getUserTags();
```

Comment:
Should this be refactored to form best practices / consistency within async/await function?